src/components/grid/Row.js

```diff
@@ -27,4 +28,4 @@ const Row = React.createClass({
   }
 });
 
-module.exports = Row;
+module.exports = Radium(Row);
```

Comment:
I know Radium doesn't play well with mobile browsers. Would this just be 'ignored'?